CHANGELOG.md

```diff
@@ -10,6 +231,7 @@
 ### Fixed
 
 - Fix ScrollViewStickyHeader to push up header above it ([d754bdefc6](https://github.com/facebook/react-native/commit/d754bdefc68ff757ac2b5a2ffa38d5aad234d484) by [@kacieb](https://github.com/kacieb))
+- Fix(deps): bump metro to 0.66.2 + dedup ([e40f58272d](https://github.com/facebook/react-native/commit/e40f58272d51a40e7b5fa77c14767ddaf9ecc006) by [@Titozzz](https://github.com/Titozzz))
```

Comment:
It looks like this line landed in `0.64.2` changelog

Reply:
Whoops! Scrolled too far, thanks for catching